system/I18n/TimeLegacy.php

```diff
@@ -47,4 +49,21 @@
 class TimeLegacy extends DateTime
 {
     use TimeTrait;
+
+    /**
+     * Returns a new instance with the date set to the new timestamp.
+     *
+     * @param int $timestamp
+     *
+     * @return static
+     *
+     * @throws Exception
```

Comment:
In the docs for the `setTimestamp` method, it mentions that an Exception may be thrown, but the method itself does not handle any exceptions directly. Should an appropriate exception be thrown?

Reply:
The parent class (`DateTimeImmutable`) throws `Exception`.

CI is kind of thin wrapper for PHP, so I don't think it's necessary to change PHP's behavior unless there is a need to do so.